lib/outh/google-outh.ts

```diff
@@ -0,0 +1,119 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const GoogleAuth = require('google-auth-library');
+const promisify = require('micro-promisify');
+const readlineSync = require('readline-sync');
+
+const { getMessage } = require('../utils/messages');
+
+import { AuthorizeCredentials } from '../../types/types';
+
+const fsReadFile = promisify(require('fs').readFile);
```

Comment:
readFileSync and writeFileSync means you can avoid the promisify/await on these.